README.rst

```diff
@@ -39,19 +39,19 @@ Sphinx Coverity plugin
 ======================
 
 Publish Coverity report along your documentation - up-to-date at every build. Include comments and other special data
-in the same table and draw some nice graphs to showcase the quality
+in the same table and draw some nice graphs to showcase the quality.
 
 ----
 Goal
 ----
 
-Coverity reporting capabilities enable you to export data into separate documentation. While this might be preferred
+Coverity's reporting capabilities enable you to export data into separate documentation. While this might be preferred,
 it is also detached from your software documentation. Another lack is that it does not include comments or any
-other special fields. That means you can end up with huge amount of intentionally triage defects, without any
-explanation why they are intentional. Because of that you cannot link your explanations for actions to that report and
+other special fields. That means you can end up with a huge amount of intentionally triaged defects, without any
+explanation why they are intentional. Because of that, you cannot link your explanations for actions to that report and
 numbers could mean just anything. Plugin should enable simple and seamless Coverity reporting integration into existing
 Sphinx documentation. Generating a reStructured Text table of defects was one option, but that allows changing before
-it is rendered, so to provide a more trustworthy path plugin that generates the data through Coverity API was
+it is rendered, so to provide a more trustworthy path plugin, that generates the data through, Coverity API was
```

Comment:
I am not sure how to fix this sentence. I don't understand the original sentence.

Reply:
Maybe better: 
```
Generating a reStructured Text table of defects was one option, but that allows changing before it is rendered, so to provide a more trustworthy path, plugin retrieves the data through Coverity API and generates/renders documentation via Sphinx without intermediate (editable) artifacts.
```

Point was that plugin does not have any intermediate files which could be edited, so it provides a chain of trust that data rendered is also data present in Coverity.

Reply:
24ff2f805b9c3789c10252c0f528d871ab0face2